graphql_compiler/tests/test_compiler.py

```diff
@@ -1151,6 +1151,147 @@ def test_in_collection_op_filter_with_optional_tag(self):
 
         check_test_data(self, test_data, expected_match, expected_gremlin)
 
+    def test_contains_op_filter_with_variable(self):
+        test_data = test_input_data.contains_op_filter_with_variable()
+
+        expected_match = '''
+            SELECT Animal___1.name AS `animal_name`
```

Comment:
Please make the formatting of all new MATCH and Gremlin queries match the formatting of the other test cases.